agent/policy.py

```diff
@@ -0,0 +1,369 @@
+from typing import List
```

Comment:
This seems to be a copy of `model.py`. Please delete whichever one is not in use.